tests/tx_generator/models/wallet_api.py

```diff
@@ -40,6 +40,9 @@ def submit_tx(self, to, src, gas_price, amount, tx_bytes):
         tx_field = '{"tx":' + str(list(tx_bytes)) + '}'
         out = self.send_api_call(pod_ip, tx_field, self.submit_api)
         print(f"{datetime.now()}: submit result: {out}")
+        if not out:
+            print("cannot parse submission result, result is none")
```

Comment:
```suggestion
            print("cannot parse submission result, result is falsy")
```
(you didn't check that it's `None`, right?)

Reply:
This is None checking.. api call will return None in case we failed

Reply:
So do
```python
if out is None:
```